cmd/server.go

```diff
@@ -157,8 +171,13 @@ func serverRun(cmd *cobra.Command, args []string) {
 			virtualServices, err := client.ListVirtualServices()
 			if err != nil {
 				log.Println("Error getting virtual services ", err)
-				os.Exit(1)
+				kempUp.Set(0)
+
+				sleep()
+				continue
 			}
+			kempUp.Set(1)
```

Comment:
Slight difference here, in that we do two network calls, so I'm setting after both of them.